### PR TITLE
Stringify webchannel event details on Thunderbird as well as Firefox

### PR DESCRIPTION
**Pull request by chenba**
Originally this was opened to run CI for #17518.  But due to time zone differences, FxA fixed the issue that was causing the test failures here.  See #17518 for its original description.

closes FXA-10411

### CODE REVIEW

**Comment by clouserw** [comment]
re-running from failed

**Comment by clouserw** [comment]
maybe a rebase?  I'm not sure why it would be timing out

File: packages/fxa-content-server/app/scripts/lib/channels/senders/web-channel.js [hunk 1]
```diff
@@ -122,8 +122,9 @@ function formatEventDetail(win, eventDetail) {
   // See https://bugzilla.mozilla.org/show_bug.cgi?id=1275616 and
   // https://bugzilla.mozilla.org/show_bug.cgi?id=1238128
   if (
-    (userAgent.isFirefoxDesktop() || userAgent.isFirefoxAndroid()) &&
-    userAgent.browser.version >= 50
```
Comment (chenba) [inline packages/fxa-content-server/app/scripts/lib/channels/senders/web-channel.js:126]: @darktrojan the test failures were caused by the removal of the browser version condition.  I've restored it to the predicate, as my attempt at updating the related code became more time consuming than I expected, and this is a quicker fix.

**Comment by chenba** [comment]
@LZoog or @clouserw r?  I need a r+ to merge

**Review by vbudhram — APPROVED** [review]
@chenba Thanks!

**Review by clouserw — APPROVED** [review]

File: packages/fxa-content-server/app/scripts/lib/channels/senders/web-channel.js (hunk 1 above)
Comment (darktrojan) [reply packages/fxa-content-server/app/scripts/lib/channels/senders/web-channel.js:126]: Really? Everything passed when I rebased it. That's weird. Oh well, as long as it works, and it appears to work.

Thanks!

**Pull request merged**